WcaOnRails/db/seeds/development/users.seeds.rb

```diff
@@ -50,3 +38,6 @@ def self.random_user
 100.times do
   FactoryGirl.create :user, :wca_id
 end
+
+# Create an admin
+FactoryGirl.create(:admin, email: "admin@wca.wca")
```

Comment:
Admin is a pretty ambiguous term for us. I always log in as `software_team`, which has more defined behavior. Is there a reason you don't sign in as `software_team`?